### PR TITLE
Add commands.help

### PR DESCRIPTION
**Pull request by windowsboy111**

### CODE REVIEW

File: guilded/ext/commands/paginator.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
+class Paginator:
+    """A class that aids in paginating code blocks for Discord messages.
```
Comment (Hidden-black) [inline guilded/ext/commands/paginator.py:3]: hmmm i thought we were using guilded

**Comment by windowsboy111** [comment]
Just looked at the request by HiddenBlack, might as well copy the whole `help.py`.

**Comment by Hidden-black** [comment]
oh ok

**Comment by windowsboy111** [comment]
Note: squashing is preferred.

**Review by Hidden-black — COMMENTED** [review]
ran through the code looks fine

**Comment by windowsboy111** [comment]
Please do test out my code, it might not work at all.

**Comment by Hidden-black** [comment]
how do i do that lmao

**Comment by Shikiiii** [comment]
> how do i do that lmao

download gpy via this pr and test it i guess

**Pull request merged**